lms/static/js/groups/views/cohorts.js

```diff
@@ -98,6 +108,28 @@ var edx = edx || {};
             this.showCohortEditor(selectedCohort);
         },
 
+        onCohortStateChanged: function(event) {
+            event.preventDefault();
+            this.saveCohortSettings();
+        },
+
+        saveCohortSettings: function() {
+            var self = this,
+                cohortSettings,
+                fieldData = {is_cohorted: this.getCohortState()};
+            cohortSettings = this.cohortSettings;
+            cohortSettings.save(
+                fieldData, {wait: true}
+            ).done(function() {
+                self.render();
+                self.renderCourseCohortSettingsNotificationView();
+            });
+        },
+
+        getCohortState: function() {
```

Comment:
I think `cohortsEnabled` in variable names would be more descriptive than `cohortState`; so we can have `getCohortsEnabled` and `onCohortsEnabledChanged`.